ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_central_germanic.txt

```diff
@@ -1384,6 +1384,20 @@
 			35 = caucasian_brown_hair
 			25 = caucasian_dark_hair
 		}
+		
+		dynasty_of_location_prefix = "dynnp_von"
```

Comment:
If we're adding this, isn't "fan" more fitting? https://en.wiktionary.org/wiki/Reconstruction:Proto-Germanic/fan%C4%93 "Von" sounds too High German for Vandals.

Reply:
I was planning to initially localize all names and dynasties, and then look back to patronymics and "de Townsville". I used _von_ as a starting fix, but I'll use _fanē_ for Vandal, _af_ (https://en.wiktionary.org/wiki/%F0%90%8C%B0%F0%90%8D%86) for Gothic.